test/presentation/rendezvous/rendezvous_list_view_model_test.dart

```diff
@@ -55,24 +55,32 @@ void main() {
 
   group('create when rendezvous state is success…', () {
     group('with rendezvous…', () {
-      test('should display them and sort them by reverse chronological order', () {
-        // Given
-        final store = _store(
-          [
-            mockRendezvous(id: '1', date: DateTime(2021, 12, 23, 10, 20)),
-            mockRendezvous(id: '2', date: DateTime(2021, 12, 24, 13, 40)),
-          ],
-        );
-
-        // When
-        final viewModel = RendezvousListViewModel.create(store);
 
-        // Then
-        expect(viewModel.displayState, DisplayState.CONTENT);
-        expect(viewModel.rendezvousIds.length, 2);
-
-        expect(viewModel.rendezvousIds[0], '2');
-        expect(viewModel.rendezvousIds[1], '1');
+      group('should display list', () {
+        final rendezvous = [
+          mockRendezvous(id: '1', date: DateTime(2021, 12, 23, 10, 20)),
+          mockRendezvous(id: '2', date: DateTime(2021, 12, 24, 13, 40)),
+        ];
+
+        test('and sort them by chronological order', () {
+          // Given
+          final store = _loggedInPEStore(rendezvous);
+          // When
+          final viewModel = RendezvousListViewModel.create(store);
+          // Then
+          expect(viewModel.displayState, DisplayState.CONTENT);
+          expect(viewModel.rendezvousIds, ['1', '2']);
+        });
+
+        test('and sort them by most recent', () {
```

Comment:
même si on comprend le test, dans le nom de la méthode j'aurais bien expliqué `and sort them by most recent for Mission Locale`

Reply:
J'ai hésité, pour ne parler que de l'intention dans le nom du test, mais plus à froid, je me dis que ça en fait peut-être parti. J'ajoute.